src/cc/tests/CMakeLists.txt

```diff
@@ -45,16 +45,31 @@ include(QfsUtil)
 set(test_sources
     integtest.cc
     integtest_main.cc
+
+    environments/MetaserverEnvironment.cc
+    environments/ChunkserverEnvironment.cc
+
```

Comment:
Yes, they are to separate out the different components. It just looked more visually pleasing. I don't hold any strong opinions for or against the empty lines though.